File: java/program75.java
import java.util.*;

class ArrayX
{
    public int Arr[];

    public ArrayX(int iSize)
    {
        this.Arr = new int[iSize];
    }

    public void Accept()
    {
        Scanner sobj = new Scanner(System.in);
        System.out.println("Enter elements of array : ");
        int iCnt = 0;

        for(iCnt = 0; iCnt < Arr.length; iCnt++)
        {
            Arr[iCnt] = sobj.nextInt();
        }
    }

    public void Display()
    {
        System.out.println("Elements of array are : ");
        int iCnt = 0;

        for(iCnt = 0; iCnt < Arr.length; iCnt++)
        {
            System.out.print(Arr[iCnt]+"\t");
        }
        System.out.println();
    }

    public float Average()
    {
        int sum = 0;
        int iCnt = 0;

        for(iCnt = 0; iCnt < Arr.length; iCnt++)
        {
            sum = sum + Arr[iCnt];
        }
        return ((float)sum / (float)Arr.length);
    }
}

class program75
{
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter number of elements : ");
        int size = sobj.nextInt();

        ArrayX obj = new ArrayX(size);
        obj.Accept();
        obj.Display();

        float fRet = obj.Average();
        System.out.println("Average is : "+fRet);
    }
}

File: cpp/program75.cpp
#include<iostream>
#include <iomanip>
using namespace std;

class ArrayX
{
    public:
        int *Arr;
        int iSize;

        ArrayX(int);
        ~ArrayX();

        void Accept();
        void Display();

        float Average();
};

ArrayX::ArrayX(int X)
{
    iSize = X;
    Arr = new int[iSize];
}

ArrayX::~ArrayX()
{
    delete []Arr;
}

void ArrayX::Accept()
{
    cout<<"Enter elements of the array \n";

    for(int iCnt = 0; iCnt < iSize; iCnt++)
    {
        cin>>Arr[iCnt];
    }
}

void ArrayX::Display()
{
    cout<<"Elements of the array are \n";

    for(int iCnt = 0; iCnt < iSize; iCnt++)
    {
        cout<<Arr[iCnt]<<"\t";
    }
    cout<<"\n";
}

float ArrayX::Average()
{
    int iSum = 0;

    for(int iCnt = 0; iCnt < iSize; iCnt++)
    {
        iSum = iSum + Arr[iCnt];
    }
    return ((float)iSum / (float)iSize);
}

int main()
{
    int iSize = 0;
    float fRet = 0.0f;

    cout<<"Enter number of elements \n";
    cin>>iSize;
 
    ArrayX obj(iSize);
    obj.Accept();
    obj.Display();

    fRet = obj.Average();
    cout<<"Average is : "<<setprecision(4)<<fRet<<"\n";

    return 0;
}